src/models/yamlProjectConfiguration.ts

```diff
@@ -197,17 +197,23 @@ export class YamlScriptsConfiguration {
     postBackendDeploy?: string;
     postFrontendDeploy?: string;
     preFrontendDeploy?: string;
+    preStartLocal?: string;
+    preReloadLocal?: string;
```

Comment:
Shouldn't we have a `postStartLocal` hook? For example, the `npm link <SDK>` command wouldn't work in a `preStartLocal` script, as the SDK is not yet generated.

Reply:
Correct.